dist/svg.js

```diff
@@ -553,16 +553,28 @@ SVG.extend(SVG.PointArray, {
     // if already is an array, no need to parse it
     if (Array.isArray(array)) return array
 
-    // split points
-    array = this.split(array)
 
     // parse points
-    for (var i = 0, il = array.length, p, points = []; i < il; i++) {
-      p = array[i].split(',')
-      points.push([parseFloat(p[0]), parseFloat(p[1])])
-    }
+    var points = array.split(/\s|,/)
+
+    // validate points - https://svgwg.org/svg2-draft/shapes.html#DataTypePoints
+    // Odd number of coordinates is an error. In such cases, drop the last odd coordinate.
+    if (points.length % 2 !== 0) points.pop()
```

Comment:
According to the [spec](https://svgwg.org/svg2-draft/shapes.html#DataTypePoints):

>If an odd number of coordinates is provided, then the element is in error, with the same user agent behavior as occurs with an incorrectly specified ‘path’ element. In such error cases the user agent will drop the last, odd coordinate and otherwise render the shape.

I think the spirit in the spec is that we should do this silently but we could easily throw an error. SVG.js should be easy to use with drawing tools such as Illustrator and perhaps throwing an error will make it more difficult for users. Someone once said be lenient with input, strict with output.